app/experimenter/nimbus-ui/src/components/TableSummary/index.tsx

```diff
@@ -56,6 +58,43 @@ const TableSummary = ({ experiment }: TableSummaryProps) => {
             )}
           </td>
         </tr>
+        <tr>
+          <th>Risk mitigation checklist</th>
+          <td data-testid="experiment-risk-mitigation-link">
+            {experiment.riskMitigationLink ? (
+              <LinkExternal href={experiment.riskMitigationLink}>
+                <span className="mr-1 align-middle">
+                  {experiment.riskMitigationLink}
+                </span>
+                <ExternalIcon />
+              </LinkExternal>
+            ) : (
+              <NotSet />
+            )}
+          </td>
+        </tr>
+        {experiment.documentationLinks &&
+          experiment.documentationLinks?.length > 0 && (
+            <tr>
+              <th>Additional links</th>
+              <td data-testid="experiment-additional-links">
+                {experiment.documentationLinks.map((documentationLink, idx) => (
+                  <React.Fragment key={`doc-link-${idx}`}>
+                    <LinkExternal
+                      href={documentationLink.link}
+                      data-testid="experiment-additional-link"
+                    >
+                      <span className="mr-1 align-middle">
+                        {documentationLink.title}
+                      </span>
+                      <ExternalIcon />
+                    </LinkExternal>
```

Comment:
> should we just move the <ExternalIcon/> reference into the ../LinkExternal.tsx definition

I wouldn't be opposed to that. What do you think @LZoog?

> I think we can possibly just define the external icon in CSS

While I do like this idea, I'd like to avoid writing custom CSS where possible (we have the bootstrap utility library). But this might be reasonable exception.

Reply:
Whatever we decide here I'll file a followup issue for.